app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/amend/check_your_answers.scala.html

```diff
@@ -59,18 +60,31 @@
     messages(messageKey)
 }
 
+@crOrPRAccountName = @{ completeReturn.representeeAnswers.fold("")(_.makeAccountName) }
+@accountName = @{ subscribedDetails.makeAccountName }
+
+@accountFullName = @{
+    completeReturn.representativeType match {
```

Comment:
cap/pr's will never be able to view the return they submitted as cap/pr's on their account home page, so we will never have representative type populated here - you could remove the cap/pr cases here and just keep the stuff under the `None` case.